controllers/topic_controller.go

```diff
@@ -33,6 +34,9 @@ const (
 	KafkaCertificate    = "KAFKA_CERTIFICATE"
 	KafkaPrivateKey     = "KAFKA_PRIVATE_KEY"
 	KafkaCA             = "KAFKA_CA"
+	KafkaStorePassword  = "KAFKA_STORE_PASSWORD"
```

Comment:
For the NAV certificates we use the variable `NAV_TRUSTSTORE_PASSWORD`. Maybe change to `KAFKA_TRUSTSTORE_PASSWORD`?

Reply:
As discussed, since this is not just for a truststore, but also for a keystore, I don't think truststore password is right.
Thoughts on `KAFKA_CREDSTORE_PASSWORD`? Short for "credentials store".

Reply:
I've renamed to `KAFKA_CREDSTORE_PASSWORD`. Also changed the keys for the stores to reflect wanted filenames instead of env-variables.